docker-compose.yml

```diff
@@ -156,6 +156,28 @@ services:
       - "traefik.frontend.rule=${TRAEFIK_FRONTEND_RULE:-}PathPrefix:/bety/"
       - "traefik.backend=bety"
 
+  # ----------------------------------------------------------------------
+  # Pecan-RStudio
+  # ----------------------------------------------------------------------
+  rstudio:
+    image: pecan/base:1.7.0
+    restart: unless-stopped
+    networks:
+      - pecan
+    environment:
+      - PASSWORD=${RSTUDIO_DEFAULT_PASSWORD:-bety}
```

Comment:
Can you add this to the env.example file. Maybe use PECAN_RSTUDIO as name for env variable, and use standard pecan password, can you set USERNAME as well?

Reply:
Added USER and appropriately named variables for both USER and PASSWORD per suggestion (in env.example as well)